README.md

```diff
@@ -52,6 +52,8 @@ RHEL: `yum install postgresql-devel`
 
 Alpine: `apk add postgresql-dev`
 
+Rename the `config/database.sample.yml` file to `config/database.yml` and edit it to match your database configuration.
```

Comment:
Using the 'cp' command to copy content to 'config/database.yml' might be better as it won't show any uncommitted changes in the git manager ?